.gitmodules

```diff
@@ -4,3 +4,7 @@
 [submodule "3rdParty/spdlog"]
 	path = 3rdParty/spdlog
 	url = https://github.com/gabime/spdlog.git
+[submodule "3rdParty/dunnart"]
+	path = 3rdParty/dunnart
+	url = https://github.com/insooth/dunnart.git
```

Comment:
Why is this is private repo link and not link to https://github.com/mjwybrow/dunnart?